.github/actions/spell-check/patterns.txt

```diff
@@ -26,6 +26,14 @@ addNSECRecordToLW.*DNSName.*powerdnt.com.*QType::NS.*res->d_records
 data:[a-zA-Z=;,/0-9+]+
 BOOST_CHECK_EQUAL\(b64, "[a-zA-Z=;,/0-9+]+"
 
+# acceptable duplicates
+# ls directory listings
+[-bcdlpsw](?:[-r][-w][-sx]){3}\s+\d+\s+(\S+)\s+\g{-1}\s+\d+\s+
+# C types
+\s(long|LONG) \g{-1}\s
```

Comment:
If `really really` is something one wanted to keep, this is the change to prevent the bot from whining about it:
```suggestion
\s(long|LONG|really) \g{-1}\s
```

Reply:
Spoke w/ @Habbie and he's ok w/ only one really, so this change isn't necessary.